p/pytest-checkdocs/pytest-checkdocs_ubi_8.4.sh

```diff
@@ -0,0 +1,62 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : pytest-checkdocs
+# Version       : v2.7.1
```

Comment:
Is this the only version being tested here ? if not, other versions tested also should be listed here

Reply:
Yes, Guru the older tags need python27 to work properly, since the build is going fine with latest tag version using python36, we are good to go, we had discussed this.